MessageMedia.Api.Console/Program.cs

```diff
@@ -114,6 +113,8 @@ public static void SendScheduledMessage()
 
                 var result = client.SendScheduledMessage("", recipient1, message, messageId, messageScheduledDateTime);
                 DisplaySendMessageResult(result);
+                
+                DeleteScheduledMessages(new List<uint>{ messageId });
```

Comment:
yeh possibly, i was being a bit lazy. Although i did update the documentation to let them know what it was doing